lib/redux/middleware/sentry_sdk_middleware.dart

```diff
@@ -21,4 +34,20 @@ class SentrySdkMiddleware extends MiddlewareClass<AppState> {
     }
     next(action);
   }
+
+  Future<void> _enableSentrySdk() async {
+    await SentryFlutter.init(
```

Comment:
initing the SDK that late, will for sure break the release health feature of this App, as a Flutter App is a single native Screen on Android and iOS, not sure if it'll report release health at all, maybe, if you put the app to the background and back to the foreground, then it works.

Reply:
Ok, how should we proceed here? Think this is a pretty common use-case nowadays to init tracking SDKs deferred and only after user consent.

What we could do is move the init out from the local storage middleware and read the consent state there before initializing the SDK. That way, it would work ass before if users gave consent. At least for app starts after users have gone once through the onboarding once.

Reply:
We can leave it as it is and deal with it later ... it's not breaking and we'll figure out how to improve it.

Reply:
testing the SDKs feature is the 1st step, and based on the results, we see what to do, but TLDR, nobody ever asked user consent for this SDK

Reply:
I created a followup task for this: https://github.com/getsentry/sentry-mobile/issues/155